locations/spiders/osservaprezzi_carburanti_it.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from scrapy import Spider
+from scrapy.http import JsonRequest, Response
+
+from locations.categories import Categories, Fuel, apply_category, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.geo import point_locations
+
+
+class OsservaprezziCarburantiITSpider(Spider):
+    name = "osservaprezzi_carburanti_it"
+
+    FUELS = {
+        "Gasolio": Fuel.DIESEL,
+        "GPL": Fuel.LPG,
+        "Metano": Fuel.CNG,
+        # "Blue Diesel": Fuel.ADBLUE, ?
+    }
+
+    BRANDS = {
+        "Api-Ip": {"brand": "IP", "brand_wikidata": "Q646807"},
+        "Esso": {"brand": "Esso", "brand_wikidata": "Q867662"},
+        "Q8": {"brand": "Q8", "brand_wikidata": "Q1634762"},
+        "Tamoil": {"brand": "Tamoil", "brand_wikidata": "Q706793"},
+        "AgipEni": {"brand": "Eni", "brand_wikidata": "Q565594"},
+        "Giap": {"brand": "Giap", "brand_wikidata": None, "nsi_id": "giap-e8f712"},
+        "Energas": {"brand": "Energas", "brand_wikidata": None, "nsi_id": "energas-e8f712"},
+        "Keropetrol": {"brand": "Keropetrol", "brand_wikidata": None, "nsi_id": "keropetrol-e8f712"},
+        "San Marco Petroli": {"brand": "San Marco Petroli", "brand_wikidata": None, "nsi_id": "sanmarcopetroli-e8f712"},
+    }
+
+    def start_requests(self):
+        for lat, lon in point_locations("italy_grid_10km.csv"):
+            payload = json.dumps(
+                {
+                    "points": [{"lat": lat, "lng": lon}],
+                    "radius": 10,
+                }
+            )
+            yield JsonRequest(
+                "https://carburanti.mise.gov.it/ospzApi/search/zone",
+                method="POST",
+                body=payload,
+            )
```

Comment:
```suggestion
            yield JsonRequest(
                "https://carburanti.mise.gov.it/ospzApi/search/zone",
                data={
                    "points": [{"lat": lat, "lng": lon}],
                    "radius": 10,
                },
            )
```

`JsonRequest` can automatically encode the data for us, and when it does, it sets the method as well.